gradle-jdks/src/main/java/com/palantir/gradle/jdks/SetupJdksTask.java

```diff
@@ -46,9 +48,29 @@ public abstract class SetupJdksTask extends DefaultTask {
     @Inject
     protected abstract ExecOperations getExecOperations();
 
+    @Inject
+    protected abstract ObjectFactory getObjectFactory();
+
+    /**
+     * If we make this an abstract getter and use @Nested, we get the following error:
```

Comment:
Note to self: In this PR, `@Nested` works for plugins but not tasks for some reason.